CHANGELOG.md

```diff
@@ -0,0 +1,50 @@
+Release Notes
+====
+
+Below you can release notes for the trunk version of WinSW.
+
+##### 2.0
+
+Release date: Coming Soon
+
+Improvements:
+* Introduce a concept of WinSW extensions, which allow extending the wrapper's behavior
+([PR #42](https://github.com/kohsuke/winsw/pull/42)).
+ * Right now the extensions are not guaranteed to be binary compatible
+  * Engine does not support their inclusion from external DLLs (but you can repackage the executable if you want to extend WinSW)
+* `SharedDirectoriesMapper` extension for mapping shared directories when the Windows service starts up 
```

Comment:
`SharedDirectoriesMapper` extension for mapping shared directories when the Windows service starts